.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+##
+## Copyright 2022 Ocean Protocol Foundation
+## SPDX-License-Identifier: Apache-2.0
+##
+name: Ocean.py multiple OS
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        version: ['3.8', '3.10', '3.11']
+    steps:
+      - name: Setup Ocean.py
+        uses: actions/checkout@v2
```

Comment:
current version is v3 https://github.com/actions/checkout

Reply:
updated, thanks!